src/lib/src/data-access/websocket/service.web.ts

```diff
@@ -14,6 +14,9 @@ export class WebSocketService<TChannelName extends string> extends BaseWebSocket
       forceTLS: this.options.useTLS,
       cluster: this.options.cluster,
       channelAuthorization: authURL ? {
+        // workaround for https://github.com/pusher/pusher-js/issues/715
```

Comment:
Do we still need this workaround?
The referenced issue is looks to be fixed and closed